src/components/space/charts-layout/ChartsArea.js

```diff
@@ -37,6 +38,9 @@ const ChartsArea = () => (
     <Grid item xs={12} sm={12} md={12} lg={12} xl={12}>
       <ItemsByActionChart />
     </Grid>
+    <Grid item xs={12} sm={12} md={12} lg={12} xl={12}>
+      <ActionsByWeekdayChart />
+    </Grid>
```

Comment:
I think this chart would be more interestin next to the `TimeOfDay` chart. Think about which position it would be best.